.circleci/config.yml

```diff
@@ -106,11 +106,14 @@ jobs:
           paths: example/node_modules
       - run:
           name: Build example
+          # setting CAPACITOR_COCOAPODS_PATH=pod makes it skip pod install.
+          # This is necessary because cocoapods isn't installed at this point.
+          # If building for iOS, it is installed and then pod install is run then.
           command: |
             cd ./example
             ./scripts/replace-placeholders.sh ${EX_APPCUES_ACCOUNT_ID} ${EX_APPCUES_APPLICATION_ID}
             npm run build
-            npx cap sync
+            CAPACITOR_COCOAPODS_PATH=pod npx cap sync
```

Comment:
interesting, so what is the expected new way of doing this then? I don't really follow how it is supposed to work without this change.

Reply:
The alternative is to have the ruby bundle installed in `example/ios/App` so that the `pod` command is available before this command is run. We could do this (I originally [tried](https://app.circleci.com/pipelines/github/appcues/appcues-capacitor-plugin/513/workflows/2441ddd0-8f05-44cf-ba02-bc7295280db3/jobs/1135?invite=true#step-102-0_36)), but I didn't seem like ruby is even installed on the `js_compatible` executor.

Reply:
ah, I think I follow now, and the way we have it set up, those next steps run on the `ios_compatible` executor.